daprdocs/content/en/java-sdk-docs/_index.md

```diff
@@ -267,5 +269,65 @@ try (DaprPreviewClient client = (new DaprClientBuilder()).buildPreviewClient())
 - For a full list of configuration operations visit [How-To: Manage configuration from a store]({{< ref howto-manage-configuration.md >}}).
 - Visit [Java SDK examples](https://github.com/dapr/java-sdk/tree/master/examples/src/main/java/io/dapr/examples/configuration) for code samples and instructions to try out different configuration operations.
 
+### Query saved state
+
+> Note this is a preview API and thus will only be accessible via the DaprPreviewClient interface and not the normal DaprClient interface
+
+```java
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+import io.dapr.client.DaprPreviewClient;
+import io.dapr.client.domain.QueryStateItem;
+import io.dapr.client.domain.QueryStateRequest;
+import io.dapr.client.domain.QueryStateResponse;
+import io.dapr.client.domain.query.Query;
+import io.dapr.client.domain.query.Sorting;
+import io.dapr.client.domain.query.filters.EqFilter;
+
+try (DaprClient client = builder.build(); DaprPreviewClient previewClient = builder.buildPreviewClient()) {
+        String searchVal = args.length == 0 ? "searchValue" : args[0];
+        
+        // Create JSON data
+        MyData dataQueried = new MyData();
+        dataQueried.setPropertyA(searchVal);
+        dataQueried.setPropertyB("query");
+        MyData dataNotQueried = new MyData();
+        dataNotQueried.setPropertyA("no query");
```

Comment:
Please, use an example that resonates better with people: Movie, Product, Car - instead of arbitrary data.

Reply:
done